src/test/java/calculator/CustomTest.java

```diff
@@ -0,0 +1,33 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.regex.Pattern;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CustomTest {
+
```

Comment:
정규식이 잘 작동하는지를 테스트하는 코드로 보여요. 이름을 바꿔 보면 어떨까요?

```suggestion

public class RegexTest {

```

Reply:
ApplicationTest 외에 제가 직접 커스텀한 테스트 코드라는 의미였는데 역시나 너무 성의 없는 네이밍이었네요... 반성하겠습니다..

Reply:
인텔리제이 쓰시면 ctrl (cmd) + shift + T 누르면 테스트 클래스를 생성할 수 있는데, 그 이름 그대로 쓰시는 것도 방법입니다!!
`InputString.class` -> `InputStringTest.class` 이렇게 만들어주더라구요 👍

Reply:
오 감사합니다!!!